cas/service.c

```diff
@@ -2317,6 +2425,7 @@ static int cas_done(struct cas_fom *fom, struct m0_cas_op *op,
 		}
 	} else
 		m0_ctg_op_fini(&fom->cf_ctg_op);
+	/* TODO: send out Persistent notices for dtm0 */
```

Comment:
Might be better to mark these places with #warning directive

Reply:
Not more relevant, removed.